packages/orchestration/src/utils/packet.js

```diff
@@ -0,0 +1,104 @@
+// @ts-check
+import { TxBody } from '@agoric/cosmic-proto/cosmos/tx/v1beta1/tx.js';
+import { Any } from '@agoric/cosmic-proto/google/protobuf/any.js';
+import { RequestQuery } from '@agoric/cosmic-proto/tendermint/abci/types.js';
+import { atob, decodeBase64, encodeBase64 } from '@endo/base64';
+import {
+  CosmosQuery,
+  CosmosResponse,
+} from '@agoric/cosmic-proto/icq/v1/packet.js';
+
+/**
+ * @import { AnyJson, RequestQueryJson, Base64Any } from '@agoric/cosmic-proto';
+ * @import { ResponseQuery } from '@agoric/cosmic-proto/tendermint/abci/types.js';
+ */
+
+/**
+ * Makes an IBC transaction packet from an array of messages. Expects the `value` of each message
+ * to be base64 encoded bytes.
+ * Skips checks for malformed messages in favor of interface guards.
+ * @param {AnyJson[]} msgs
+ * @param {Partial<Omit<TxBody, 'messages'>>} [opts]
+ * @returns {string} - IBC TX packet
+ * @throws {Error} if malformed messages are provided
+ */
+export function makeTxPacket(msgs, opts) {
+  const messages = msgs.map(Any.fromJSON);
+  const bytes = TxBody.encode(
+    TxBody.fromPartial({
+      messages,
+      ...opts,
+    }),
+  ).finish();
+
+  return JSON.stringify({
+    type: 1,
```

Comment:
magic number. please import or name it with a const

Reply:
Referencing `InterchainPacketData` and and the `Type.TYPE_EXECUTE_TX` enum now: [cb1a005](https://github.com/Agoric/agoric-sdk/pull/9198/commits/cb1a005f2d63b81e0a39773c98cb4e2f2fa1c315)